cmd/preflight/cmd/check_container.go

```diff
@@ -210,28 +225,41 @@ func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 		}
 	})
 
-	// --submit was specified
 	viper := viper.Instance()
+
+	// If the new flag is set, use that
+	if cmd.Flag("certification-component-id").Changed {
+		cmd.Flag("certification-project-id").Changed = true
+		cmd.Flag("certification-component-id").Changed = false
+		viper.Set("certification_project_id", componentID)
+	}
```

Comment:
If I'm understanding, this is just  "promotion" of the value from one flag to the other, right? And updating their Changed state is required to avoid MutualExclusion from throwing an error?

Semantic & Nit: it reads weird to me that we're feeding the value of the new flag into the old flag. Disregard at will.

Reply:
It is awkward, but this makes it so that we don't have to go change a bunch of other places that use the project-id verbage. It does seem very awkward. 

It occurs to me now to that each of these places that are changed should be marked with a TODO so that they are easier to find later.